brave-core/src/main/java/com/github/kristofa/brave/TracerAdapter.java

```diff
@@ -1,6 +1,7 @@
 package com.github.kristofa.brave;
 
 import brave.Tracer;
+import brave.Tracing;
```

Comment:
I'm guessing you backed out some changes in this file. Probably don't need the import